### PR TITLE
add function to generate random points within a polygon

### PR DESCRIPTION
**Pull request by karpfen**
I closed the last pull request because I found a couple errors. Fixed now!

### CODE REVIEW

**Comment by mpadge** [comment]
@karpfen you don't have to close a PR - just fix the errors and it'll automatically be updated to reflect that

**Comment by mpadge** [comment]
Plus maybe just change the `sp::bbox()` calls for simpler `xmin <- min(poly_city[, 1])`, etc. Other than that, all looks grand - thanks!

**Comment by karpfen** [comment]
Oh cool, didn't know that. Updated now!

File: R/get-random-points.R [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+#'
+#' @param xmin Minimum x coordinate
+#' @param ymin Minimum y coordinate
+#' @param xmax Maximum x coordinate
+#' @param ymax Maximum y coordinate
+#' @param n Number of coordinates to be generated
+#' @return \code{matrix} with two columns of length n with random coordinates
+#'
+#' @noRd
+random_points_in_polygon <- function (xmin, ymin, xdiff, ydiff, n)
```
Comment (mdsumner) [inline R/get-random-points.R]: looks like this should be

```R
random_points_in_polygon <- function (xmin, ymin, xmax, ymax, n)
```

Comment (karpfen) [inline R/get-random-points.R]: Thanks! Must have missed that somehow.

**Pull request merged**